### DISCUSSION TITLE
Docker image?

### DISCUSSION THREAD

**Discussion by zephiowl** [Cog-Creators/Red-DiscordBot#5742] [Q&A] [answered]
Hey is there a docker image for red? if so where is I cant seem to find it on hub.docker.com

**Comment by JeffJrShim** [comment, marked as answer]
Docker isn't supported by Red.

**Reply by zephiowl** [reply]
Ahh i see; do you know if its in the pipeline at all? most services host things with docker; from home servers to azure and everything else. It'd be a good way to host the bot

**Reply by JeffJrShim** [reply]
There's something in progress, but I'm not sure when it will be finished, and I don't think its supported either. (https://github.com/Cog-Creators/Red-Docker)

**Reply by zephiowl** [reply]
Ahh i see; well i really hope we get an official version; i was able to find a third party one that seems to work just fine for what i need. 

https://hub.docker.com/r/phasecorex/red-discordbot

**Comment by zephiowl** [comment]
For anyone else that may find this in the future; there is currently no official version; but I have found a updated third party docker image which will work just as good here. 

https://hub.docker.com/r/phasecorex/red-discordbot
